soh/soh/Enhancements/randomizer/3drando/hint_list.cpp

```diff
@@ -2714,6 +2714,13 @@ void HintTable_Init() {
               /*spanish*/ "Los sabios aguardarán a que el héroe&obtenga #%d |símbolo|símbolos| de&skulltula dorada#." },
     });
 
+    hintTable[BRIDGE_GREG_HINT] = HintText::Bridge({
+        // obscure text
+        Text{ "The awakened ones will await&for the Hero to find %gGreg%w.",
+              /*french*/ "The awakened ones will await&for the Hero to find %gGreg%w.",
```

Comment:
Do we need some sort of TODO comment here at least?

Reply:
made an issue for greg translations and assigned @PurpleHato https://github.com/HarbourMasters/Shipwright/issues/2533